assets/RNAseq_report.Rmd

```diff
@@ -31,7 +31,23 @@ params:
 df <- read.csv(file = params$path_proj_summary, sep="\t", header = TRUE)
 # Import the .tsv file downloadable from https://portal.qbic.uni-tuebingen.de/portal/web/qbic/user-database -> Projects
 projectcode = as.character(df$Sub.Project)
+# Set the title; if the title is too long, it would get covered by the logo, so it is split in 2 or 3 parts.
 title = as.character(df$Short.Title)
+if (nchar(title) > 65 & nchar(title) < 130) {
```

Comment:
Think about this if clause: if the number of characters is exactly 65 or 130 then it goes to the else clause as well. It would be nice to have here an option that prints any string lengths into chunks of 65. Even if they are longer than say 200 characters. Could you think of a solution here?

Reply:
it works now :)

Reply:
This is still not ideal:
- it does not work for a title where nchar(title) = 130
- it does not produce a title for nchar(title) < 65

Reply:
I updated it, [see](https://github.com/silviamorins/rnadeseq/blob/d31a59e7791e37de5199832f10b46ea40ca92d27/assets/RNAseq_report.Rmd#L41)